src/Catel.MVVM/Catel.MVVM.Shared/MVVM/ViewModels/ViewModelMetadata.cs

```diff
@@ -25,8 +25,8 @@ internal class ViewModelMetadata
         /// <exception cref="ArgumentNullException">The <paramref name="models"/> is <c>null</c>.</exception>
         /// <exception cref="ArgumentNullException">The <paramref name="mappings"/> is <c>null</c>.</exception>
         /// <exception cref="ArgumentNullException">The <paramref name="validations"/> is <c>null</c>.</exception>
-        internal ViewModelMetadata(Type viewModelType, Dictionary<string, ModelInfo> models, Dictionary<string, ViewModelToModelMapping> mappings,
-            Dictionary<string, ValidationToViewModelAttribute> validations)
+        internal ViewModelMetadata(Type viewModelType, IReadOnlyDictionary<string, ModelInfo> models, IReadOnlyDictionary<string, ViewModelToModelMapping> mappings,
```

Comment:
I have done it to ensure that this class will be immutable, but this is internal class so I will restore it with ReadonlyDictionary class.